packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -458,6 +461,32 @@ async function _getDashboardStats(app) {
     console.error(error)
   }
 
+  let nConnectedServers = 0
+  let nUnreachableServers = 0
+  let nUnknownServers = 0
+  servers.forEach(server => {
+    if (server.status === 'connected' && poolIds.includes(server.poolId)) {
```

Comment:
```suggestion
// some servers can be connected with unknown pool
    if (server.status === 'connected' && poolIds.includes(server.poolId)) {
```

Reply:
I created a card in `intakes` about that

Reply:
ok for me, you can close this comment when @b-Nollet is ok